packages/block-renderer/src/hooks/use-pattern-min-height-vh.ts

```diff
@@ -0,0 +1,18 @@
+import { useMemo } from 'react';
+
+const usePatternMinHeightVh = ( html = '', viewportHeight: number | undefined = 0 ) => {
+	return useMemo( () => {
+		return html.replace( /min-height:\s?(?<value>\d+)vh;?/g, ( match, value ) => {
```

Comment:
@arthur791004 thanks for the suggestion. I had to replace `\s+` with `\s?` to make it work without spaces.

Reply:
Oh yes, `\s?` is correct one 🙈